package.json

```diff
@@ -1,17 +1,13 @@
 {
-  "name": "hello-world-lumen",
+  "name": "hello-world-lume",
   "version": "1.0.0",
   "description": "",
   "main": "_config.js",
   "scripts": {
-    "build": "deno task lume",
-    "start": "serve _site/"
```

Comment:
We aim to make our starters work with both Static Site Hosting and Application Hosting. App hosting requires `serve` to work correctly.

But you are right - if not the App hosting, it wouldn't be required